terraform/README.md

```diff
@@ -95,25 +119,20 @@ For more information check the Makefile contents.
 
 ## Run bare Terraform commands
 
-You have to use the terraform wrapper script to run terraform commands directly:
+Generally you won't need to run bare Terraform commands as our ```Makefile``` should cover most needed scenarios. However, if you do then you should use the terraform wrapper script to run them directly. The terraform wrapper script takes care of loading your ```.envrc``` and running ```aws-auth```.
 
-```
-cd root/aws-dm
-./../../terraform-wrapper output
-```
-
-This command would have the same effect:
+For example, if you wanted to run ```terraform output``` against preview you can do so:
 
 ```
-cd root/aws-dm
-. .envrc && aws-auth terraform output
+cd environments/preview
+./../../terraform-wrapper output
 ```
 
-## Requirements in a new AWS environment
+## Requirements in a clean or completely new AWS environment
```

Comment:
Is the clean-ness here something about the remote environment, or something about the checkout? I don't quite follow when we are supposed to need these instructions.

Reply:
So I think this is the situation where we have no existing AWS infrastructure.

E.g. we have no s3 buckets that store our terraform state, we have no users.

This is obviously not the situation that we are currently in (and unlikely to be in?).

I can either remove this section on the assumption that it will never be useful or I can give a better explanation. I'm leaning towards the later. Opinions?

Reply:
Ah, okay yeah maybe "How we created the AWS environment".

(Trouble I suppose is: is that true? Or is this an idealised version of how we might have created it if we hadn't have been iterating something that existed already? Or is it just how we _might_ create it if we needed to - and is that a realistic thing that could happen if somehow our AWS infrastructure got wiped out by an extreme data fail scenario?)